src/org/openbravo/erpCommon/ad_process/HeartbeatProcess.java

```diff
@@ -4,48 +4,56 @@
  * Version  1.1  (the  "License"),  being   the  Mozilla   Public  License
  * Version 1.1  with a permitted attribution clause; you may not  use this
  * file except in compliance with the License. You  may  obtain  a copy of
- * the License at http://www.openbravo.com/legal/license.html 
+ * the License at http://www.openbravo.com/legal/license.html
  * Software distributed under the License  is  distributed  on  an "AS IS"
  * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
  * License for the specific  language  governing  rights  and  limitations
- * under the License. 
- * The Original Code is Openbravo ERP. 
- * The Initial Developer of the Original Code is Openbravo SLU 
- * All portions are Copyright (C) 2008-2020 Openbravo SLU 
- * All Rights Reserved. 
+ * under the License.
```

Comment:
These license comments must have been modified when running a complete refactorization of the class. Normally in Etendo we try to not change them at all. Try reverting these changes made by the refactorization